### DISCUSSION TITLE
Innr SP 120 off after reboot

### DISCUSSION THREAD

**Discussion by salopette** [Koenkk/zigbee2mqtt#5305] [Devices]
Hello, I have 2 Innr SP 120 sockets.

with one the problem is that it always switches off after a restart of z2m. The other one doesn't do it, she always stays in the same state.
Can you stop that? I've only seen this feature on Ikea Plug.

**Comment by Koenkk** [comment]
Can you provide the debug log when starting Zigbee2MQTT, starting at the start of Zigbee2MQTT until the switch turns off?

To enable debug logging set in `configuration.yaml`:
```yaml
advanced:
  log_level: debug
```

**Comment by salopette** [comment]
`Dez 15 08:23:38 raspberrypi npm[10786]: Zigbee2MQTT:debug 2020-12-15 08:23:38: Received Zigbe                                                                                                e message from 'Keller-St-Licht', type 'attributeReport', cluster 'haElectricalMeasurement',                                                                                                 data '{"rmsCurrent":0,"rmsVoltage":235}' from endpoint 1 with groupID 0
Dez 15 08:23:38 raspberrypi npm[10786]: Zigbee2MQTT:info  2020-12-15 08:23:38: MQTT publish:                                                                                                 topic 'zigbee2mqtt/Keller-St-Licht', payload '{"current":0,"energy":707,"last_seen":"2020-12-                                                                                                15T07:23:38.256Z","linkquality":48,"power":36,"state":"OFF","voltage":235}'
Dez 15 08:23:41 raspberrypi npm[10786]: Zigbee2MQTT:debug 2020-12-15 08:23:41: Received Zigbe                                                                                                e message from 'Keller-St-Licht', type 'attributeReport', cluster 'haElectricalMeasurement',                                                                                                 data '{"activePower":0}' from endpoint 1 with groupID 0
Dez 15 08:23:41 raspberrypi npm[10786]: Zigbee2MQTT:info  2020-12-15 08:23:41: MQTT publish:                                                                                                 topic 'zigbee2mqtt/Keller-St-Licht', payload '{"current":0,"energy":707,"last_seen":"2020-12-                                                                                                15T07:23:41.263Z","linkquality":45,"power":0,"state":"OFF","voltage":235}'
Dez 15 08:23:46 raspberrypi npm[10786]: Zigbee2MQTT:debug 2020-12-15 08:23:46: Received Zigbee message from 'Lade Plug', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsCurrent":58}' from endpoint 1 with groupID 0
Dez 15 08:23:46 raspberrypi npm[10786]: Zigbee2MQTT:info  2020-12-15 08:23:46: MQTT publish: topic 'zigbee2mqtt/Lade Plug', payload '{"current":0.06,"energy":156,"last_seen":"2020-12-15T07:23:46.258Z","linkquality":24,"power":6,"state":"ON","voltage":227}'
Dez 15 08:23:51 raspberrypi npm[10786]: Zigbee2MQTT:debug 2020-12-15 08:23:51: Received Zigbee message from 'Lade Plug', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsCurrent":63}' from endpoint 1 with groupID 0
Dez 15 08:23:51 raspberrypi npm[10786]: Zigbee2MQTT:info  2020-12-15 08:23:51: MQTT publish: topic 'zigbee2mqtt/Lade Plug', payload '{"current":0.06,"energy":156,"last_seen":"2020-12-15T07:23:51.177Z","linkquality":27,"power":6,"state":"ON","voltage":227}'
`

**Comment by salopette** [comment]
`debug 2020-12-15 08:23:36: Received Zigbee message from 'Keller-St-Licht', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"activePower":36}' from endpoint 1 with groupID 0
info  2020-12-15 08:23:36: MQTT publish: topic 'zigbee2mqtt/Keller-St-Licht', payload '{"current":0.27,"energy":707,"last_seen":"2020-12-15T07:23:36.850Z","linkquality":51,"power":36,"state":"ON","voltage":235}'
info  2020-12-15 08:23:36: MQTT publish: topic 'zigbee2mqtt/Keller-St-Licht', payload '{"current":0.27,"energy":707,"last_seen":"2020-12-15T07:23:36.850Z","linkquality":51,"power":36,"state":"OFF","voltage":235}'
info  2020-12-15 08:23:37: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"commit":"6198460","coordinator":{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20201026,"transportrev":2},"type":"zStack3x0"},"log_level":"debug","network":{"channel":15,"extendedPanID":"0xdddddddddddddddd","panID":10597},"permit_join":false,"version":"1.16.2-dev"}'
debug 2020-12-15 08:23:37: Received Zigbee message from 'Keller-St-Licht', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2020-12-15 08:23:37: MQTT publish: topic 'zigbee2mqtt/Keller-St-Licht', payload '{"current":0.27,"energy":707,"last_seen":"2020-12-15T07:23:37.246Z","linkquality":45,"power":36,"state":"OFF","voltage":235}'
`

**Comment by salopette** [comment]
Here is the whole log!

https://cloud.mail.ru/public/Euba%2F5rHRFArwC

**Reply by Koenkk** [reply]
As can be seen in the log Zigbee2MQTT receives a request to turn off this device: 

```
debug 2020-12-15 08:23:36: Received MQTT message on 'zigbee2mqtt/Keller-St-Licht/set' with data '{"current":0.27,"energy":573,"last_seen":"2020-12-07T16:01:50.816Z","linkquality":90,"power":39,"state":"OFF","voltage":230}'
```

Likely this is a retained MQTT message (https://community.openhab.org/t/clearing-mqtt-retained-messages/58221) and you have to clear it, or it could be send by e.g. an automation. Anyway this is not a Zigbee2MQTT issue.

**Comment by salopette** [comment]
Thanks, delete mqtt .db worked!
